domains/transfer-agreements/TransferAgreements.API/API.UnitTests/Controllers/TransferAgreementsControllerTests.cs

```diff
@@ -55,4 +61,37 @@ public async Task Create_ShouldCallServiceOnce()
             agreement.ReceiverTin == request.ReceiverTin
         )), Times.Once);
     }
+
+    [Fact]
+    public async Task GetBySubjectId_ShouldCallServiceOnce()
```

Comment:
I would suggest the names of all the test indicates that this is a list and not `GetBySubjectId`
eg. `GetTransferAgreements_`